packages/vendor-core/scss/variables.scss

```diff
@@ -1,5 +1,9 @@
 /**
-  Third Party / Dependency imports here
+  Third Party variables imports here
 */
+$font-path: '~patternfly/dist/fonts/';
+$img-path: '~patternfly/dist/img/';
+$icon-font-path: '~patternfly/dist/fonts/';
```

Comment:
IIUC this is overriding the variables from patternfly, shouldn't it be done after importing them?

Reply:
It should be done before because patternfly uses the scss `!default` feature.

Reply:
see https://sass-lang.com/documentation/variables#default-values

Reply:
nice, learned something new

Reply:
btw it is done so solve this issue:
```scss
// lib.scss
$color: yellow;
$border: 1px solid $color;

// theme
@import 'lib';
$color: yellow;
```

will not affect the $border in this case